packages/musap-react-native/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@sphereon/musap-react-native",
+  "title": "musap-rn",
+  "version": "0.0.0",
+  "description": "A React Native module wrapping the MUSAP library for Android & iOS",
+  "homepage": "https://github.com/Sphereon-Opensource/musap-react-native",
+  "main": "dist/index.js",
+  "module": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "nativePackage": true,
+  "files": [
+    "src",
+    "dist",
+    "android/{src,maven,build.gradle,gradle.properties,settings.gradle}",
+    "ios/{Podfile,Podfile.lock,.xcode.env}",
+    "ios/musap-react-native",
+    "expo-module.config.json",
+    "musap-react-native.podspec"
+  ],
+  "scripts": {
+    "build": "tsc --build --clean && tsc --build",
+    "prepare": "yarn build",
+    "build:android:windows": "cd android && gradlew.bat publishReleaseAar",
+    "build:android:linux": "cd android && ./gradlew publishReleaseAar",
+    "build:ios": "cd ios && xcodebuild -scheme MusapRn -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 12' build"
+  },
+  "installConfig": {
+    "hoistingLimits": "workspaces"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Sphereon-Opensource/musap-react-native.git",
+    "baseUrl": "https://github.com/Sphereon-Opensource/musap-react-native"
+  },
+  "keywords": [
+    "react-native",
+    "musap"
+  ],
+  "author": {
+    "name": "Sphereon",
+    "email": "dev@sphereon.com"
+  },
+  "license": "Apache-2.0",
+  "licenseFilename": "LICENSE",
+  "readmeFilename": "README.md",
+  "dependencies": {
+    "@sphereon/musap-native": "workspace:*",
+    "@sphereon/musap-react-native": "0.0.1-unstable.89+5df316cc",
```

Comment:
Why isn't this workspace:*?

Reply:
No that is its own package, removed. (This also explains why I had to set an override in SSI-SDK-crypto-extensions)